projects/plugins/jetpack/extensions/shared/plan-utils.js

```diff
@@ -31,7 +31,12 @@ import { requiresPaidPlan } from './register-jetpack-block';
  */
 export function getUpgradeUrl( { planSlug, plan, postId, postType } ) {
 	// WP.com plan objects have a dedicated `path_slug` field, Jetpack plan objects don't.
-	const planPathSlug = startsWith( planSlug, 'jetpack_' ) ? planSlug : get( plan, [ 'path_slug' ] );
+	let planPathSlug = startsWith( planSlug, 'jetpack_' ) ? planSlug : get( plan, [ 'path_slug' ] );
+
+	// Overrites path to `business` for Atomic sites.
+	if ( isAtomicSite() ) {
```

Comment:
Is this `if` statement still needed now that we're redirecting to the plans page?

Reply:
nop 🤦 removing...